### DISCUSSION TITLE
Fainted Code copying button after purchasing licence

### DISCUSSION THREAD

**Discussion by NajimSaleh** [primefaces/community#469] [PrimeVue Blocks] [answered]
I have just purchased a PrimeBlock license and I expected to unlock the "Code" button that's on the top right corner of each block for easy copying of the block's code but surprisingly that button is still fainted !! Does this mean the only way to get a block's code is to login and download the whole PrimeBlock code base per version ???

**Comment by tugcekucukoglu** [comment, marked as answer]
Yes, you could download the latest version from PrimeStore and then run locally.

**Reply by v-dale** [reply]
we need to be able to use it online. why should I run it locally?

**Comment by betterbydesign** [comment]
I just purchased this too but I still don't understand how to unlock the copy/paste option here:
![CleanShot 2023-12-29 at 12 24 44@2x](https://github.com/primefaces/community/assets/3317473/9ae7d7b4-09ee-44eb-a4f8-230ea7ebfcf7)

**Reply by cantin-f** [reply]
You need to download the project and run it locally. The button will be available.

I know it sucks...

I've seen Prime work on make it available online.
